script.js

```diff
@@ -1,4 +1,180 @@
+      " VS Computer's score: " +
+      CompNumOfCorrectGuess +
+      "<br> Now you can type scissors, paper or stone to play another round!✨"
+    );
+  }
+
+  if (randomObject == 1) {
+    console.log(paper);
+    console.log("paper runs");
+    if (userVComputer(input, "✋")) {
+      if (input == scissors) {
+        UserNumOfCorrectGuess += 1;
+        console.log(UserNumOfCorrectGuess);
+      }
+      return (
+        "You win!💯💯<br> You chose " +
+        input +
+        ", while the computer chose ✋ <br> You: " +
+        UserNumOfCorrectGuess +
+        " VS Computer: " +
+        CompNumOfCorrectGuess +
+        "<br> Now you can type scissors, paper or stone to play another round!✨"
+      );
+    }
+
+    if (input == "paper") {
+      return (
+        "It's a draw...?👉👈<br> You chose " +
+        input +
+        ", while the computer chose ✋ <br> You: " +
+        UserNumOfCorrectGuess +
+        " VS Computer: " +
+        CompNumOfCorrectGuess +
+        "<br> Now you can type scissors, paper or stone to play another round!✨"
+      );
+    }
+
+    if (input == stone) {
+      CompNumOfCorrectGuess += 1;
+      console.log(CompNumOfCorrectGuess);
+    }
+    return (
+      "You lose...😧😧<br> You chose " +
+      input +
+      ", while the computer chose ✋ <br> You: " +
+      UserNumOfCorrectGuess +
+      " VS Computer: " +
+      CompNumOfCorrectGuess +
+      "<br> Now you can type scissors, paper or stone to play another round!✨"
+    );
+  }
+
+  if (randomObject != 0 || randomObject != 1) {
+    console.log(stone);
+    console.log("stone runs");
+    if (input == paper) {
+      UserNumOfCorrectGuess += 1;
+      console.log(UserNumOfCorrectGuess);
+    }
+    return (
+      "You win!💯💯<br> You chose " +
+      input +
+      ", while the computer chose ✊ <br> You: " +
+      UserNumOfCorrectGuess +
+      " VS Computer: " +
+      CompNumOfCorrectGuess +
+      "<br> Now you can type scissors, paper or stone to play another round!✨"
+    );
+  }
+
+  if (input == "stone") {
+    return (
+      "It's a draw...?👉👈<br> You chose " +
+      input +
+      ", while the computer chose ✊ <br> You: " +
+      UserNumOfCorrectGuess +
+      " VS Computer: " +
+      CompNumOfCorrectGuess +
+      "<br> Now you can type scissors, paper or stone to play another round!✨"
+    );
+  }
+
+  if (input == scissors) {
+    CompNumOfOfCorrectGuess += 1;
+    console.log(CompNumOfCorrectGuess);
+  }
+  return (
+    "You lose...😧😧<br> You chose " +
+    input +
+    ", while the computer chose ✊ <br> You: " +
+    UserNumOfCorrectGuess +
+    " VS Computer: " +
+    CompNumOfCorrectGuess +
+    "<br> Now you can type scissors, paper or stone to play another round!✨"
+  );
+};
+
+var userVComputer = function (input, ComputerHand) {
+  return (
+    (input == "scissors" && ComputerHand == "paper") ||
+    (input == "paper" && ComputerHand == "stone") ||
+    (input == "stone" && ComputerHand == "scissors")
+  );
+};
```

Comment:
Great use of a helper function here. A slight improvement would be the naming of the function. Try to name your functions starting with a verb. An example would be : isWinner